diagnostics/conf/routes

```diff
@@ -14,3 +14,4 @@ GET        /counts.gif                      controllers.DiagnosticsController.an
 GET        /robots.txt                      controllers.Assets.at(path="/public", file="robots.txt")
 
 POST       /css                             controllers.DiagnosticsController.css
+OPTIONS    /css                             controllers.DiagnosticsController.cssOptions
```

Comment:
Yes, it could be. we would have to be very lenient with the response though, eg. we would have to permit all methods (GET, POST, OPTIONS), even if we didn't support them for the particular endpoint. I want to check this actually does help things in prod first, then we can find a less tedious way of dealing with Cors.